lib/reunion.rb

```diff
@@ -0,0 +1,31 @@
+class Reunion
+  attr_reader :name,
+              :activities
+
+  def initialize(name)
+    @name = name
+    @activities = []
+  end
+
+  def add_activity(activity)
+    @activities << activity
+  end
+
+  def total_cost
+    @activities.sum {|activity| activity.total_cost}
+  end
+
+  def breakout
```

Comment:
This will _only_ work with two activities 😱 You need to always think about scalability when implementing functionality!